ci-operator/config/openshift/installer/openshift-installer-master.yaml

```
@@ -380,6 +380,24 @@ tests:
     cluster_profile: aws
     workflow: openshift-e2e-aws-upi-proxy
   timeout: 6h0m0s
+- always_run: false
+  as: e2e-aws-user-provisioned-dns
+  optional: true
+  steps:
+    cluster_profile: aws
+    env:
+      FEATURE_SET: TechPreviewNoUpgrade
+      USER_PROVISIONED_DNS: "yes"
+    steps:
+    cluster_profile: aws
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-conf-aws
+    - chain: ipi-install
+    test:
+    - ref: openshift-e2e-test
```

Do you have to create a custom workflow for this? Can't you just do
```suggestion
    cluster_profile: aws
    env:
      FEATURE_SET: TechPreviewNoUpgrade
      USER_PROVISIONED_DNS: "yes"
    workflow: openshift-e2e-aws
```